src/CategoriesContainer.jsx

```diff
@@ -1,40 +1,41 @@
 import { useDispatch, useSelector } from 'react-redux';
 
-import {
-  selectCategory,
-  loadRestaurants,
-} from './actions';
+import { selectCategory } from './redux_modules/restaurantSlice';
+import { loadRestaurants } from './redux_modules/asyncActions';
 
-import { get } from './utils';
+import MenuList from './MenuList';
+import MenuItem from './MenuItem';
+import SelectButton from './SelectButton';
 
 export default function CategoriesContainer() {
   const dispatch = useDispatch();
 
-  const categories = useSelector(get('categories'));
-  const selectedCategory = useSelector(get('selectedCategory'));
+  const categories = useSelector((state) => state.restaurant.categories);
+  const selectedCategory = useSelector((state) => state.restaurant.selectedCategory);
 
   function handleClick(categoryId) {
     dispatch(selectCategory(categoryId));
     dispatch(loadRestaurants());
   }
 
   return (
-    <ul>
-      {categories.map((category) => (
-        <li key={category.id}>
-          <button
-            type="button"
-            onClick={() => handleClick(category.id)}
+    <MenuList>
+      {categories.map((category) => {
+        const isSelected = selectedCategory && (category.id === selectedCategory.id);
```

Comment:
category의 id가 undefined가 아니라는 가정하에 간단하게 `optional chaining`을 이용해서 아래처럼 표현해줄 수도 있을 것 같네요
```suggestion
        const isSelected = selectedCategory?.id === category.id;
```

Reply:
옵셔널체이닝을 이런 상황에서 적용할 수 있군요. 개념 자체는 알았는데 사용경험이 딱히 없다보니 생각을 못했네요. 덕분에 좋은 경험 얻어갑니다~